.github/workflows/checks.yml

```diff
@@ -26,7 +26,7 @@ jobs:
           echo "::set-output name=branchBaseCommit::$(git merge-base origin/develop HEAD)"
       - name: Diff-Checkstyle
         run: |
-          curl -s -L https://github.com/yangziwen/diff-checkstyle/releases/download/0.0.4/diff-checkstyle.jar > diff-checkstyle.jar
+          curl -s -L https://github.com/yangziwen/diff-check/releases/download/0.0.7/diff-checkstyle.jar > diff-checkstyle.jar
```

Comment:
Did you check whether this still catches checkstyle violations on CI?

Reply:
Yes, I tested it.
Some notes: The actual code is almost exactly the same. The only difference is how the repo is organized. See here:
Before: https://github.com/yangziwen/diff-checkstyle/tree/master/src/main/java/io/github/yangziwen/checkstyle
After: https://github.com/yangziwen/diff-check/tree/master/diff-checkstyle/src/main/java/io/github/yangziwen/checkstyle

In the after, the `diff` subdirectory is located in a different module: https://github.com/yangziwen/diff-check/tree/master/diff-core/src/main/java/io/github/yangziwen/diff/calculate